src/test/java/shouty/CoordinateTest.java

```diff
@@ -0,0 +1,14 @@
+package shouty;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+
+public class CoordinateTest extends TestCase {
+
+    @Test
+    public void testItCalculatesTheDistanceFromAnotherCoordinateAlongXAxis() {
```

Comment:
select the project root or src/test dir. Then cmd-shift-F10 IIRC. or use the mouse.